tests/testsuite/publish_lockfile.rs

```diff
@@ -249,7 +249,7 @@ fn note_resolve_changes() {
 [NOTE] package `multi v0.1.0` added to the packaged Cargo.lock file, was originally sourced from `[..]/foo/multi`
 [NOTE] package `patched v1.0.0` added to the packaged Cargo.lock file, was originally sourced from `[..]/foo/patched`
 [PACKAGED] [..] files, [..] ([..] compressed)
-[WARNING] no (git) Cargo.toml found at `target/tmp/[..]/foo/Cargo.toml` in workdir `[..]`
```

Comment:
Is this something we're going to regularly run into?  Is there an auto-redactuion we can use to help avoid this?

Reply:
It is a bit special because it points to the path of our test infra, not the test case environment. I don't think it is a common thing.